### PR TITLE
Feat: Manage Menu Group added to Booking Detail screen

### PR DESCRIPTION
**Pull request by RobinCsl**
Summary: I also prepared screens for the corresponding routes. The route configurations might need to be improved, especially when I suspect that the booking id might be necessary for the Share Booking screen for instance.

### CODE REVIEW

File: apps/core/scenes/bookingDetail/BookingDetail.js [hunk 1]
```diff
@@ -13,7 +16,12 @@ type Props = {|
 
 export default class BookingDetail extends React.Component<Props> {
   renderInner = (props: BookingDetailQueryResponse) => {
-    return <TripDetails data={props.bookingDetail} />;
+    return (
+      <ScrollView contentContainerStyle={styles.container}>
```
Comment (tbergq) [inline apps/core/scenes/bookingDetail/BookingDetail.js]: I think we should move this to a separate component, unless this renderInner will turn into a beast

File: apps/core/scenes/bookingDetail/menuGroups/Manage.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      }
+      case 'android': {
+        shareIcon = 'share-android';
+        break;
+      }
+      default: {
+        shareIcon = 'share';
+        break;
+      }
+    }
```
Comment (tbergq) [inline apps/core/scenes/bookingDetail/menuGroups/Manage.js]: How about creating a ShareIcon component ? We will use this logic also in boarding pass component

File: apps/web/pages/mmb/other.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const ManageOtherScreen = () => (
+  <Layout>
+    <View style={styles.wrapper}>
+      <ManageOther />
+    </View>
+  </Layout>
+);
+
+const styles = StyleSheet.create({
+  wrapper: {
```
Comment (tbergq) [inline apps/web/pages/mmb/other.js]: Should we consider moving this to layout component?

Comment (RobinCsl) [reply apps/web/pages/mmb/other.js]: I moved it to a local Layout component; it seems specific to MMB views for now.

**Comment by RobinCsl** [comment]
@tbergq 🚀

**Pull request merged**